src/main/java/net/mcreator/ui/dialogs/preferences/PluginsPanel.java

```diff
@@ -110,7 +110,7 @@ public Component getListCellRendererComponent(JList<? extends Plugin> list, Plug
 
 			if (value.getInfo().getAuthor() != null)
 				setText("<html>" + value.getInfo().getName() + "<br><small>ID: " + value.getID() + ", version: " + value
-						.getPluginVersion() + ", author: " + value.getInfo().getAuthor());
+						.getPluginVersion() + ", author: " + value.getInfo().getAuthor() + ", credit: " + value.getInfo().getCredit());

```

Comment:
Add another check if credit is null, so null will not be shown when the credit field is empty

Reply:
I already added a check to replace "null" by "None".

Reply:
It would be better to add another field and omit None which would be shown in all other cases I think